tests/e2e_regression/eden/expected/emerald_token_nfts_eth.headers

```diff
@@ -1,4 +1,4 @@
-HTTP/1.1 404 Not Found
+HTTP/1.1 200 OK

```

Comment:
Note to self/readers: This went 404->200 because even though it's the same endpoint, we now run it in eden, but we used to run it in damask: https://github.com/oasisprotocol/nexus/pull/675#discussion_r1538516256